virtualization/windowscontainers/management/container_networking.md

```diff
@@ -214,8 +216,37 @@ C:\> docker network inspect <network name>
 ### Multiple Container Networks
 
 Multiple container networks can be created on a single container host with the following caveats:
-* Only one NAT network can be created per container host.
+
 * Multiple networks which use an external vSwitch for connectivity (e.g. Transparent, L2 Bridge, L2 Transparent) must each use its own network adapter.
+* Currently, our solution for creating multiple NAT networks on a single container host is to subnet the NAT network IP address block. Reference the below section, 'Multiple NAT Networks' for further guidance on this.
```

Comment:
These smaller partitions must be within the the larger NAT internal which can be found by referencing the "InternalIPInterfaceAddressPrefix" field when running Get-NetNat from PowerShell or the "Subnet" field when running "docker network inspect nat". Feel free to word-smith this.